index.js

```diff
@@ -36,7 +28,7 @@
      */
     checkComponentVisibility: function() {
       var domnode = this._dom_node,
-          gcs = getComputedStyle(domnode, false),
+          gcs = window.getComputedStyle(domnode, false),
```

Comment:
Yes it's needed for the tests. It's best not to rely on window being global because it doesn't work well with jsdom. It's also easier to read! (at first, I couldn't work out what we were trying to getComputedStyle *of*!)